packages/server/src/api/controllers/table/utils.ts

```diff
@@ -496,5 +497,31 @@ export function setStaticSchemas(datasource: Datasource, table: Table) {
   return table
 }
 
+export async function internalTableCleanup(table: Table, rows?: Row[]) {
+  const db = context.getAppDB()
+  const tableId = table._id!
+  // remove table search index
+  if (!env.isTest() || env.COUCH_DB_URL) {
```

Comment:
Shouldn't this be checking if there sqs is enabled, instead of `env.COUCH_DB_URL`?

Reply:
What is the reason why this is excluded from tests?